verification/curator-service/ui/src/components/new-case-form-fields/Events.tsx

```diff
@@ -27,11 +27,13 @@ export default function Events(): JSX.Element {
                 <DateField
                     name="confirmedDate"
                     label="Confirmed case date"
+                    required={true}
```

Comment:
nit. can we just say `required` ? if no no big deal.

Reply:
Done